Formula/kyma-cli.rb

```diff
@@ -0,0 +1,36 @@
+class KymaCli < Formula
+  desc "Kyma command-line interface"
+  homepage "https://kyma-project.io"
+  url "https://github.com/kyma-project/cli.git",
+      :tag      => "1.5.0",
+      :revision => "6c8148df282e0bd23a377e9e5eb250c33179a1ca"
+  head "https://github.com/kyma-project/cli.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    bin_path = buildpath/"src/github.com/kyma-project/cli/"
+    bin_path.install Dir["*"]
+
+    cd bin_path do
+      system "dep", "ensure", "-vendor-only"
+      system "make", "build-darwin"
+      bin.install "bin/kyma-darwin"
```

Comment:
This can have a second parameter specifying the new name that would make the line below redundant.

Reply:
Done. Thanks for the tip!